handyrl/train.py

```diff
@@ -163,7 +164,14 @@ def forward_prediction(model, hidden, batch, args):
                 hidden_ = map_r(hidden_, lambda h: h.sum(1))  # (..., B * 1, ...)
             else:
                 hidden_ = map_r(hidden_, lambda h: h.view(-1, *h.size()[2:]))  # (..., B * P, ...)
-            outputs_ = model(obs, hidden_)
+            if t < args['burn_in_steps']:
+                model.eval()
+                with torch.no_grad():
+                    outputs_= model(obs, hidden_)
+            else:
+                if not model.training:
+                    model.train()
+                outputs_ = model(obs, hidden_)
             for k, o in outputs_.items():
                 if k == 'hidden':
                     next_hidden = outputs_['hidden']
```

Comment:
BTW, this can be `next_hidden = o` ? I don't care that though...

Reply:
I belive so. I will create chore PR.